packages/rum-core/src/domain/rumEventsCollection/action/getSelectorsFromElement.ts

```diff
@@ -31,24 +31,33 @@ export function getSelectorsFromElement(element: Element, actionNameAttribute: s
   return {
     selector: getSelectorFromElement(
       element,
-      attributeSelectors.concat(getIDSelector),
+      attributeSelectors.concat(createIDSelector({})),
       attributeSelectors.concat(createClassSelector({}))
     ),
     selector_without_classes: getSelectorFromElement(
       element,
-      attributeSelectors.concat(getIDSelector),
+      attributeSelectors.concat(createIDSelector({})),
       attributeSelectors
     ),
     selector_without_body_classes: getSelectorFromElement(
       element,
-      attributeSelectors.concat(getIDSelector),
+      attributeSelectors.concat(createIDSelector({})),
       attributeSelectors.concat(createClassSelector({ ignoreBody: true }))
     ),
+    selector_without_generated_id_and_classes: getSelectorFromElement(
+      element,
+      attributeSelectors.concat(createIDSelector({ ignoreGeneratedValue: true })),
+      attributeSelectors.concat(createClassSelector({ ignoreGeneratedValue: true }))
+    ),
   }
 }
 
 type GetSelector = (element: Element) => string | undefined
 
+function isGeneratedValue(value: string) {
+  return /[0-9]/.test(value)
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could be nice to add a comment to explain the reasoning